src/Microsoft.PowerShell.Commands.Utility/commands/utility/Get-Error.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Management.Automation;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// Class for Get-Error implementation.
+    /// </summary>
+    [Experimental("Microsoft.PowerShell.Utility.PSGetError", ExperimentAction.Show)]
+    [Cmdlet(VerbsCommon.Get, "Error", HelpUri = "https://docs.microsoft.com/powershell/module/microsoft.powershell.utility/get-error?view=powershell-7&WT.mc_id=ps-gethelp", DefaultParameterSetName = NewestParameterSetName)]
```

Comment:
Minor nitpick: Long lines like this are very hard to read. how about wrapping the values of the `Cmdlet` attribute across multiple lines?

Reply:
will update